packages/frontity-org-theme/src/processors/border-radius.ts

```diff
@@ -0,0 +1,32 @@
+import { css } from "frontity";
+
+const createBorderRadiusProcessors = (options: {
+  [className: string]: ReturnType<typeof css>;
+}) => {
+  return Object.entries(options).map(([className, style]) => ({
+    name: className,
+    test: ({ node }) =>
+      node.type === "element" &&
+      node.props.className &&
+      node.props.className.split(" ").includes(className),
+    processor: ({ node }) => ({
+      ...node,
+      props: {
+        ...node.props,
+        css: css`
+          ${node.props.css}
+          ${style}
+        `
+      }
+    })
+  }));
+};
+
+export default createBorderRadiusProcessors({
+  "has-border-radius-8px": css`
```

Comment:
Hey, maybe we can change this to get the number of pixels from the class name? I guess that's the idea so we could change the number of pixels in the future just changing that value in the class.

Reply:
Thanks for the feedback @DAreRodz.

The issue with this is that we might complicate the test function for the processor. For your implementation to work, we'll need a global class `has-border-radius` for all elements that have border-radius and another class with the size of the border-radius.

Reply:
@SantosGuillamot What do you think?

Reply:
I think what David was proposing was to keep using the same class, `has-border-radius-XXpx` and get the pixels with a regexp. This way we will be using just one class but it'd be more flexible so we can use it for other `border-radius` like `has-border-radius-16px`.

Reply:
So, we will have 2 classes for elements with border-radius. One will be `has-border-radius` and the other will be `has-border-radius-XXpx`.

I'm gonna go update the classes on the wp-admin dashboard.